Modules/_json.c

```diff
@@ -1356,6 +1356,32 @@ _steal_accumulate(_PyUnicodeWriter *writer, PyObject *stolen)
     return rval;
 }
 
+// TODO: this method could be cached
+PyObject* _create_newline_indent(PyObject* indent, Py_ssize_t indent_level)
+{
+    PyObject* start = PyUnicode_FromString("\n");
+    PyObject* newline_indent = NULL;
+    if (start == 0) {
+        goto end;
```

Comment:
`mul_name` will be used uninitialized if this branch is taken.

Also, `== 0` looks odd for pointer comparisons; compare against NULL instead.